src/pages/workspace/WorkspaceSidebar.js

```diff
@@ -58,39 +58,64 @@ const defaultProps = {
     allPolicies: null,
 };
 
+/**
+ * @param {Object} navigationState
+ * @returns {String|undefined}
+ */
+function getPolicyIDFromNavigationState(navigationState) {
+    const routes = lodashGet(navigationState, 'routes', []);
+    const routeWithPolicyIDParam = _.find(routes, route => route.params && route.params.policyID);
+    return lodashGet(routeWithPolicyIDParam, ['params', 'policyID']);
+}
+
 const WorkspaceSidebar = ({
-    translate, isSmallScreenWidth, policy, allPolicies, isFocused,
+    translate, isSmallScreenWidth, policy, allPolicies, isFocused, navigation,
 }) => {
+    const policyID = lodashGet(policy, 'id');
     const menuItems = [
         {
             translationKey: 'workspace.common.card',
             icon: ExpensifyCard,
             action: () => {
-                Navigation.navigate(ROUTES.getWorkspaceCardRoute(policy.id));
+                if (policyID) {
```

Comment:
+1 although I'm guessing since we are fixing a race condition or something that it will be clickable a short time after.

Another option would be to wait until we have a policy to show these options at all.

Reply:
> Another option would be to wait until we have a policy to show these options at all.

This seems like a better solution. Or we could make them appear disabled if we don't have a policy ID. As it was before, clicking these would attempt to navigate to `workspace//card` or `workspace//people`. Choosing between linking to those and doing nothing (aka, what we have here), I would suggest doing nothing.